posthog/clickhouse/migrations/0098_sessions_v2_irclid_kx.py

```diff
@@ -0,0 +1,23 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.models.raw_sessions.migrations import (
+    WRITABLE_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL,
+    DISTRIBUTED_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL,
+    BASE_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL,
+)
+from posthog.models.raw_sessions.sql import (
+    DROP_RAW_SESSION_MATERIALIZED_VIEW_SQL,
+    RAW_SESSIONS_TABLE_MV_SQL,
+    RAW_SESSIONS_VIEW_SQL,
+)
+
+operations = [
+    # drop the mv, so we are no longer receiving events from the sessions table
+    run_sql_with_exceptions(DROP_RAW_SESSION_MATERIALIZED_VIEW_SQL()),
+    # now we can alter the target tables
+    run_sql_with_exceptions(WRITABLE_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL()),
+    run_sql_with_exceptions(DISTRIBUTED_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL()),
+    run_sql_with_exceptions(BASE_RAW_SESSIONS_ADD_VITALS_LCP_COLUMN_SQL()),
+    # and then recreate the materialized view and view
+    run_sql_with_exceptions(RAW_SESSIONS_TABLE_MV_SQL()),
+    run_sql_with_exceptions(RAW_SESSIONS_VIEW_SQL()),
```

Comment:
Ah... I'd probably rename this "CREATE_OR_REPLACE_RAW_SESSIONS_VIEW_SQL" to help the future traveller

threw me that we were creating something we hadn't dropped

(but don't optimise for my brain)

Reply:
Done!